typos.toml

```diff
@@ -0,0 +1,20 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
+[files]
+extend-exclude = [
+    "vendor/*",
+    "third_party/*",
+    "*generated.pb.go",
+    "*.svg",
+    "*CHANGELOG*",
+    "*.drawio",
+]
+[default.extend-words]
+Ded = "Ded"
+Datas = "Datas"
+unexpect = "unexpect"
+Mis = "Mis"
```

Comment:
By the way.
```
error: `Mis` should be `Miss`, `Mist`
  --> ./pkg/util/selector.go:38:10
   |
38 |  PriorityMisMatch ImplicitPriority = iota
```
This might not be a false-positive, as `mismatch` is a valid word. 
I think we can replace `PriorityMisMatch` with `PriorityMismatch`.

Reply:
That's true, and there's the tradeoff: we can mark the invalid words as "valid" words if the correction will bring some break changes or something like that. And when it's okay to do that, we can remove the mark and fix them all.